forem/forem
# [15-minute fix] Disable 'Locked Screen' prompt for public Forems

## What type of PR is this? (check all applicable)


## Description

This PR removes the "Use as 'Locked Screen'" prompt from the admin area of public Forems. The original PR description suggested disabling the checkbox but since our admin area is already a bit busy I suggested completely hiding the option instead at it seems general consensus was in favor of this suggestion.

## Related Tickets & Documents

https://github.com/forem/forem/issues/13964

## QA Instructions, Screenshots, Recordings

1. Visit `/admin/customization/pages`
2. Clock "Edit" on one of your pages
3. You should not see a prompt to set the page as "Locked screen"
    ![image](https://user-images.githubusercontent.com/47985/122717186-7633d300-d295-11eb-827c-e384e102cfb1.png)
4. Set the Forem to private (e.g. `Settings::Authentication.invite_only_mode = true` in a Rails console)
5. Reload the page
6.  You should see a prompt to set the page as "Locked screen"
    ![image](https://user-images.githubusercontent.com/47985/122717292-9c597300-d295-11eb-8aa8-f82bc1f806ac.png)

### UI accessibility concerns?

None

## Added/updated tests?


## [Forem core team only] How will this change be communicated?
